### ISSUE TITLE
Unknown Issues

### ISSUE THREAD

**Issue by jingyetan** [NVlabs/stylegan3#82]
Dear StyleGANs3 developers

I am attempting to use your library and my first step was to try to use one of the pre-trained network example.  However, was calling the generator, it keeps getting stuck at this message:

> Loading networks from "https://api.ngc.nvidia.com/v2/models/nvidia/research/stylegan3/versions/1/files/stylegan3-r-afhqv2-512x512.pkl"...
> Generating image for seed 2 (0/1) ...
> Setting up PyTorch plugin "bias_act_plugin"... 

I have the current version of your library, running on Linux CentOS with miniconda, the environment is created by the .yml file in your library, and also loaded cuda/11.3.1 and gcc/10.2.0.  The GPU is NVDIA tesla v100.

Could you please help me out to get this issue resolved?  Thank you!

**Comment by nurpax** [comment]
This step can take some time to complete (1-2 minutes).  But there's a catch: if you ctrl-c out of it, PyTorch may leave a lock file under the torch extensions dir which will cause all further attempts to always hang on this step.

Please try if any of the tips on https://github.com/NVlabs/stylegan3/blob/main/docs/troubleshooting.md help.

**Comment by nuclearsugar** [comment]
Thank you for the tip. This fixed it for me:

> Completely remove: $HOME/.cache/torch_extensions (Linux) or C:\Users\<username>\AppData\Local\torch_extensions\torch_extensions\Cache (Windows) and re-run StyleGAN3 python code.
